src/Symfony/Component/DependencyInjection/Compiler/MergeExtensionConfigurationPass.php

```diff
@@ -47,6 +48,9 @@ public function process(ContainerBuilder $container)
             $tmpContainer = new ContainerBuilder($container->getParameterBag());
             $tmpContainer->setResourceTracking($container->isTrackingResources());
             $tmpContainer->addObjectResource($extension);
+            if ($extension instanceof ConfigurationExtensionInterface && null !== $configuration = $extension->getConfiguration($config, $tmpContainer)) {
+                $tmpContainer->addObjectResource($configuration);
```

Comment:
Correct :) see https://github.com/symfony/symfony/pull/21057#discussion_r93932858

I first targeted master, doing it only here (with in mind we do a changelog on behavioral change). 

However now we target 2.7, in order to fix the behavior in lower branches. Indeed giving multiple resources by default. But's not really a big a deal, only weird looking at it :)